### ISSUE TITLE
Output of the inpainter gives a gray boundary

### ISSUE THREAD

**Issue by chirag126** [DAA233/generative-inpainting-pytorch#5]
Most of the generated images from running "python test_tf_model.py --image examples/imagenet/imagenet_patches_ILSVRC2012_val_00008210_input.png --mask examples/center_mask_256.png --output examples/output.png --model-path torch_model.p" gives a constant gray boundary for all inputs.
![output](https://user-images.githubusercontent.com/12678048/60140006-b844df80-9775-11e9-858e-a21d60096139.png)

**Comment by DAA233** [comment]
Yes, the results are not very good when testing on the converted TF model.

The TF model was converted by semi-manually matching the weight/shapes/names between the TensorFlow variables and PyTorch model weights according to [here](https://github.com/DAA233/generative-inpainting-pytorch/pull/2#issuecomment-489389673). I don't know if there are some incompatible part. And there are also some operations that don't have weights like contextual attention, which may incompatible with the original one.

**Comment by chirag126** [comment]
Thanks for the information. In that case, do you also have the pre-trained weights for the Pytorch model?

**Comment by DAA233** [comment]
Yes, I just updated the PyTorch model download links in the README. Now you can download and test on it.

**Comment by chirag126** [comment]
Perfect, thanks. I will give it a try.

**Comment by chirag126** [comment]
The pre-trained model weights are giving size mismatch errors for different fine_generator layers. Any comment?

**Comment by DAA233** [comment]
Em... so embarrassing. 

The pre-trained model weights are trained with a bug version --- some convolution layers' channel numbers are wrong in that version. The bugs were fixed by [this pull request](https://github.com/DAA233/generative-inpainting-pytorch/pull/2/commits/12f49c441edcdf585905f1758f6b5a2f14cb8d86#diff-584f4b3ab12058252e96596e3fd01fffL102). So you need to change some channel numbers to load the model. Sorry about those problems.

After fixing the bugs, I haven't trained the model again. I prepare to train a model from scratch later but it will cost a long time.

**Comment by chirag126** [comment]
Yes, I already did that but was hoping that you have a pre-trained version of the original model.

> --
> Regards,
> Chirag Agarwal
> UIN: 670916062
> PhD Student, ECE Department
> University of Illinois at Chicago.

**Comment by DAA233** [comment]
@chirag126 New model released with commit b2d7de.
